packages/calypso-products/src/plans-list.tsx

```diff
@@ -1437,6 +1438,7 @@ const getPlanPremiumDetails = (): IncompleteWPcomPlan => ( {
 		FEATURE_VIDEOPRESS_JP,
 		FEATURE_UNLTD_SOCIAL_MEDIA_JP,
 		FEATURE_WORDADS,
+		FEATURE_STATS_PAID,
```

Comment:
This is the link-in-bio flow.
<img width="1439" alt="link-in-bio" src="https://github.com/Automattic/wp-calypso/assets/140841/8b7b2991-5d9d-4881-a33c-78a4841afd34">